bela-belasobral93/Homework-1/temp-converter/js/main.js

```diff
@@ -0,0 +1,68 @@
+/*
+1. Create a function that accepts a temperature value in Fahrenheit and converts it to Celsius.
+   Note: Equation to convert F to C:
+   C = (F - 32) / 1.8
+   Test your function.
+*/
+
+
+ //return first element in document with id "temperature", in this case, the user input
+
+let celcius;
+let temp = document.querySelector("#temperature").value;
+
+   function fToC(){
+      return celcius = (temp - 32) / 1.8;
```

Comment:
You want to try to make sure that your return statement is the last thing that runs in your function. If you return before you console log it wont ever get to that step